.github/workflows/reusable-test.yml

```diff
@@ -100,7 +100,7 @@ jobs:
 
     # Perform shallow checkout for self-hosted runner.
     - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
-      if: (inputs.environment == 'ebpf_cicd_tests' || inputs.environment == 'ebpf_cicd_tests_ws2022' || inputs.environment == 'ebpf_cicd_perf') && (steps.skip_check.outputs.should_skip != 'true')
+      if: (inputs.environment == 'ebpf_cicd_tests_ws2019' || inputs.environment == 'ebpf_cicd_tests_ws2022' || inputs.environment == 'ebpf_cicd_perf') && (steps.skip_check.outputs.should_skip != 'true')
```

Comment:
Question: is ebpf_cicd_perf 2019 or 2022?

Reply:
2022

Reply:
I'll rename that runner too. Next scheduled run is only in the evening.